refactor!: expose connectionIndicator as constant instead of via getter

joheriks opened the pull request:
Fixes #9671

haijian-vaadin reviewed (approved):
The code looks good to me. 
Just a comment regarding the commit message, would `refactor` be better than `fix` for the code change?

joheriks commented:
> The code looks good to me.
> Just a comment regarding the commit message, would `refactor` be better than `fix` for the code change?

Updated. Also fixed relying on accessor for side effect (adding indicator to DOM), as this could be optimized away (and so happened in `test-root-context` ITs).
